### PR TITLE
Drupal 9 compatibility fixes

### PR DESCRIPTION
**Pull request by bbenbahloul**
### Description

Fix Drupal 9 compatibility issues.

### Change log

- Added:
- Changed: update all module files to specify that the module is compatible with drupal 9
- Deprecated: replace deprecated link function with toLink
- Removed:
- Fixed:
- Security:

### CODE REVIEW

File: tests/Functional/EntityMetaRelationContentFormTest.php [hunk 1]
```diff
@@ -89,8 +94,8 @@ public function testContentWithSingleEntityMetaEditing(): void {
     $this->getSession()->getPage()->fillField('Title', $label);
     $this->getSession()->getPage()->uncheckField('Published');
     $this->getSession()->getPage()->fillField('Color', 'red');
-    $this->assertFalse($this->getSession()->getPage()->hasField('Volume'));
-    $this->assertFalse($this->getSession()->getPage()->hasField('Gear'));
+    $this->assertFalse((boolean) $this->getSession()->getPage()->hasField('Volume'));
```
Comment (yenyasinn) [inline tests/Functional/EntityMetaRelationContentFormTest.php]: Drupal core uses `(bool)` for boolean casting. Let's do it also for consistency.

File: src/Entity/EntityMeta.php [hunk 2]
```diff
@@ -51,6 +51,7 @@
  *     "status" = "status"
  *   },
  *   revision_metadata_keys = {
+ *     "revision_user" = "revision_user",
```
Comment (yenyasinn) [inline src/Entity/EntityMeta.php:54]: Explanation for next reviewers: https://www.drupal.org/node/2831499.

File: runner.yml.dist [hunk 3]
```diff
@@ -32,7 +32,7 @@ drupal:
 
 commands:
   drupal:site-setup:
-    - { task: "symlink", from: "../../..", to: "${drupal.root}/modules/custom/emr" }
+    - { task: "run", command: "drupal:symlink-project" }
```
Comment (yenyasinn) [inline runner.yml.dist:35]: Please add to the README.md file informatation about drupal:symlink-project command. Check https://github.com/openeuropa/oe_media/pull/163/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R69, https://github.com/openeuropa/multivalue_form_element/pull/9/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R36, https://github.com/openeuropa/oe_multilingual/pull/121/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R69.

File: composer.json [hunk 4]
```diff
@@ -6,31 +6,23 @@
...
-        "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/task-runner": "~1.0.0-beta5",
-        "phpunit/phpunit": "~6.0",
-        "symfony/browser-kit": "~3.0||~4.0"
+        "drush/drush": "~10.3",
+        "openeuropa/code-review": "~1.6",
+        "openeuropa/drupal-core-require-dev": "^8.9 || ^9.1",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
+        "phpspec/prophecy-phpunit": "^1 || ^2",
+        "phpunit/phpunit": "^6.5 || ^7 || ^8 || ^9"
```
Comment (yenyasinn) [inline composer.json]: We don't need to define `"phpspec/prophecy-phpunit": "^1 || ^2",` and ` "phpunit/phpunit": "^6.5 || ^7 || ^8 || ^9"` both. Leave just one option.

**Review by upchuk — APPROVED** [review]

**Review by hernani — APPROVED** [review]

**Pull request merged**